mlpipeline/src/mlpipeline/ingredient_parser/advanced_parser.py

```
@@ -0,0 +1,187 @@
+                ] + few_shot_examples + [
+                    {'role': 'user', 'content': f"Extract from: {text}"}
+                ],
+                format=IngredientParsed.model_json_schema(),
+                options={
+                    'temperature': 0, 
+                    'num_ctx': 1024,
+                    'num_predict': 128
+                }
+            )
+            return IngredientParsed.model_validate_json(response.message.content)
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)